sensors/src/main/scala/com/lynbrookrobotics/potassium/sensors/imu/ADIS16448.scala

```diff
@@ -0,0 +1,19 @@
+package com.lynbrookrobotics.potassium.sensors.imu
+
+import com.lynbrookrobotics.potassium.sensors.SPI
+import squants.motion.AngularVelocity
+
+/**
+  * An interface for communicating with the ADIS16448 IMU.
+  */
+class ADIS16448(spi: SPI) extends DigitalGyro {
+
+    private val imuCom: ADIS16448Protocol = new ADIS16448Protocol(spi)
```

Comment:
Because the logic that goes into DigitalGyro is so small, just move everything from `ADIS16448Protocol` into here.